Sources/StreamChat/Utils/Logger/Logger.swift

```diff
@@ -178,20 +178,26 @@ public enum LogConfig {
     /// Underlying logger instance to control singleton.
     private static var _logger: Logger?
 
+    private static var queue = DispatchQueue(label: "io.getstream.logconfig")
```

Comment:
Isn't it better for it to be concurrent with a barrier? (`attributes: .concurrent`)

Reply:
That is for performance reasons. I think that, in this case, the optimization is not necessary at all 🤔

Reply:
I've made it concurrent 👍 Since this is a logger, it probably makes sense to make sure it does not cause performance issues. Although it would be very unlikely